OrbitGl/GraphTrack.h

```diff
@@ -0,0 +1,28 @@
+#ifndef ORBIT_GL_GRAPH_TRACK_H
+#define ORBIT_GL_GRAPH_TRACK_H
+
+#include "ScopeTimer.h"
+#include "Track.h"
+
+#include <limits>
+
+class TimeGraph;
+
```

Comment:
This should have a comment explaining usage.

Reply:
Will do in future PR when introducing variable tracking feature.